docs/public/specifications/tests/Zone-TP/zone11.md

```diff
@@ -39,15 +63,18 @@ server.
 
 ## Summary
 
-Message Tag                      | Level   | Arguments          | Message ID for message tag
-:--------------------------------|:--------|:-------------------|:--------------------------------------------
-Z11_INCONSISTENT_SPF_POLICIES    | WARNING |                    | One or more name servers do not publish the same SPF policy as the others.
-Z11_DIFFERENT_SPF_POLICIES_FOUND | NOTICE  | ns_ip_list         | The following name servers returned the same SPF policy, but other name servers returned a different policy. Name servers: {ns_ip_list}.
-Z11_NO_SPF_FOUND                 | NOTICE  | domain             | No SPF policy was found for {domain}.
-Z11_SPF1_MULTIPLE_RECORDS        | ERROR   | ns_ip_list         | The following name servers returned more than one SPF policy. Name servers: {ns_ip_list}.
-Z11_SPF1_SYNTAX_ERROR            | ERROR   | domain, ns_ip_list | The SPF policy of {domain} has a syntax error. Policy retrieved from the following nameservers: {ns_ip_list}.
-Z11_SPF1_SYNTAX_OK               | INFO    | domain             | The SPF policy of {domain} has correct syntax.
-Z11_UNABLE_TO_CHECK_FOR_SPF      | ERROR   |                    | None of the zone’s name servers responded with an authoritative response to queries for SPF policies.
+Message Tag                      | Level   | Arguments       | Message ID for message tag
+:--------------------------------|:--------|:----------------|:--------------------------------------------
+Z11_DIFFERENT_SPF_POLICIES_FOUND | NOTICE  | ns_list         | The following name servers returned the same SPF policy, but other name servers returned a different policy. Name servers: {ns_list}.
```

Comment:
Z11_DIFFERENT_SPF_POLICIES_FOUND is always outputted together with Z11_INCONSISTENT_SPF_POLICIES. The message ID could the be shortened to `The following name servers returned the same SPF policy. Name servers: {ns_list}.`